addons/mail/static/src/models/device/device.js

```diff
@@ -21,7 +21,7 @@ function factory(dependencies) {
          * @override
          */
         _willDelete() {
-            window.removeEventListener('resize', this._onResize);
+            this.messaging.browser.removeEventListener('resize', this._onResize);
```

Comment:
Is there a reason we need the indirection of messaging instead of importing browser where we need it?

Reply:
- no need to worry about import
- lesson learned from env, I don't want diff in the code everywhere again the next time someone decide to move browser somewhere else in the web client